packages/toCamelCase/package.json

```diff
@@ -0,0 +1,34 @@
+{
+  "name": "@plexis/to-camel-case",
+  "version": "0.0.12",
+  "description": "Convert string to camel case",
+  "keywords": [
+    "camel",
+    "case",
+    "camelcase"
+  ],
+  "author": "Andres Ruiz-Velediaz <aruiz858@gmail.com>",
+  "homepage": "https://github.com/Mercyfulsin/plexis#readme",
+  "license": "ISC",
```

Comment:
Hey, the license is wrong as well as some keys for the package. 
You can create the new package using the template from here https://github.com/plexis-js/plexis/tree/master/.package-template